parlai/core/fixed_data_teacher.py

```diff
@@ -4,14 +4,88 @@
 # LICENSE file in the root directory of this source tree. An additional grant
 # of patent rights can be found in the PATENTS file in the same directory.
 
-from .agents import Teacher
+from .agents import Teacher, create_task_agent_from_taskname
 
+import argparse
+from collections import deque
 import concurrent.futures
 from threading import Thread
 import queue
 import random
 
 
+def flatten(teacher, context_length=None, include_label=True):
```

Comment:
i feel like these should be in some utils file or our teachers file is going to be hard to read and full of stuff? (also kurt is moving this file)

Reply:
sure!